src/lib/misc.js

```diff
@@ -10,7 +10,7 @@ const escodegen = require('escodegen')
 const Errors = require('@oclif/errors')
 var rp = require('request-promise')
 
-const FAUNA_CLOUD_DOMAIN = 'db.fauna.com'
+const FAUNA_CLOUD_DOMAIN = 'db.fauna-preview.com'
```

Comment:
When running `cloud-login`, the `validCloudEndpoint()` function checks to see if this string is in the URL. If not, it logs the error on line 15. The error doesn't show up the first time you configure the `cloud` endpoint, but will be displayed any subsequent time `cloud-login` is run and the endpoint setup will fail.